scalableminds/webknossos
# second try: Remove Volume Version-Freezing code

this time with front-end

### URL of deployed dev instance (used for testing):
- https://nomorefreezeversions.webknossos.xyz

### Steps to test:
- trace some  (skeleton + volume)
- open version history view
- should not show errors

------
